Agreements/Internal/TCWEL_28.txt

```diff
@@ -0,0 +1,42 @@
+COPX:EGKK:*:BEDFO:*:*:London TC WELIN:London S28:21000:*:^BEDFO
+
+COPX:EGKK:*:WELIN:*:*:London TC WELIN:London S28:21000:*:^WELIN
+
+COPX:EGLL:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
```

Comment:
Are these not climbing agreements?

Reply:
No, level by agreements to avoid SE TRENT

Reply:
Ah, had thought that would just be implied, given that a/c shouldn't enter a third sector without coordination